data-generation/src/main/scala/utils/UtilsUtils.scala

```diff
@@ -59,4 +65,33 @@ object UtilsUtils {
       case None => UniformType(0, 0)
     }
   }
+
+  def persistSomeDistributions() = {
+    getConfig("ENV_VARIABLES_ARE_SET") // Use this line if you want to make sure that env variabels are set
+    implicit val spark: SparkSession = getSparkSession
+    implicit val sc: SparkContext = spark.sparkContext
+
+    val logger: Logger = LoggerFactory.getLogger(getClass.getSimpleName)
+
+    val param1 = UtilsUtils.getConfig("DISTRIBUTION_PARAM1").toDouble
+    val param2 = UtilsUtils.getConfig("DISTRIBUTION_PARAM2").toDouble
+
+    val distributionType = UtilsUtils.loadDistributionType()
+    val distribution = (iteration: Int) => distributionType match {
+      case _: LogNormalType => LogNormalType(param1.toInt, iteration * param2)
+      case _: GaussianType => GaussianType(param1.toInt, iteration * param2)
+      case _: UniformType => UniformType(param1, iteration * param2)
+      case _: ZipfType => ZipfType(param1.toInt, iteration * param2)
```

Comment:
```suggestion
      case _: LogNormalType => LogNormalType(param1.toInt, iteration * param2)
      case _: GaussianType => GaussianType(iteration * param1.toInt, param2)
      case _: UniformType => UniformType(param1, iteration * param2)
      case _: ZipfType => ZipfType(param1.toInt, iteration * param2)
```
For Gaussian så endrer det andre parameteret variansen, så om vi ønsker varierende expected logs burde vi endre til noe á la dette

Reply:
ja, skal fikse det begge steder 👍